transient/cli.py

```diff
@@ -4,6 +4,7 @@
 import signal
 import sys
 
+from . import config as _config
```

Comment:
I assume this is to avoid the module getting shadowed by a local 'config' variable? I'd rather change the module to 'configuration' or something if this is a common issue.

Reply:
I debated this a bit. Your assumption is correct; it's used to prevent shadowing from local config variables.

I could name the module `configuration.py` but it's asymmetric with `transient/utils.py` and `config/`.

Currently, this module is only used in `cli.py` and `transient.py`.

I don't have a predilection to either - I just wanted to try to keep symmetry with the existing codebase.

Reply:
I needed to add more public members to `config.py` so I ended up renaming this file to `configurations` so `_config` isn't in multiple places.